demo/pkg/injector/context.go

```diff
@@ -0,0 +1,33 @@
+package injector
```

Comment:
The name is very generic. Can it be more specific? What does it inject?

Reply:
It injects HTTP headers and initial WS payload into the context.Context so the schema resolvers can read that information (gqlgen doesn't pass it). What name do you have in mind?

Reply:
Fine for me. I missed the context. I comment why we created this package would be great.

Reply:
I've just added a doc.go file to the package with a bit of context https://github.com/wundergraph/cosmo/pull/212/files#diff-b671d1b1d70e1a60cf24e556532a391eeafa95b8fd18c59cea390153f4670250